### DISCUSSION TITLE
Example Plugin

### DISCUSSION THREAD

**Discussion by Tiebe** [Impostor/Impostor#213] [Q&A] [answered]
Hey,

Could someone give me a zip with a simple example plugin? I can't get it to work.

Thanks in advance,

Tiebe

**Comment by Mm2PL** [comment, marked as answer]
There is an example plugin in the source files. You can browse the files [here.](https://github.com/Impostor/Impostor/tree/dev/src/Impostor.Plugins.Example)

**Comment by Tiebe** [comment]
Thank you!

**Comment by Tiebe** [comment]
When I copy that folder and I try to build it, I get the following errors:

![errors](https://user-images.githubusercontent.com/9307389/101291500-958f7500-3809-11eb-8c56-25ffd6029019.png)

**Comment by Tiebe** [comment]
@Mm2PL

**Comment by Tiebe** [comment]
Nevermind, I didn't have the latest version of the Imposter.Api
